cypress/support/step_definition/room_boards/commonRoomBoardSteps.js

```diff
@@ -0,0 +1,9 @@
+const { When, Then } = require("@badeball/cypress-cucumber-preprocessor");
```

Comment:
file name is not correct and cucumber is not detecting it although tests are passing

Reply:
this is like this already in many other step definition files too, here are some more example:
\support\step_definition\account\accountSettingsSteps.spec.js
\support\step_definition\account\commonAccountSteps.spec.js
....... etc.

this would be a bigger clean up follow up then, out of scope.

Reply:
done!